pageserver/client_grpc/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "pageserver_client_grpc"
+version = "0.1.0"
+edition = "2024"
+
+[dependencies]
+bytes.workspace = true
+futures.workspace = true
+http.workspace = true
+thiserror.workspace = true
+tonic.workspace = true
+tracing.workspace = true
+pageserver_page_api.workspace = true
+utils.workspace = true
+tokio.workspace = true
+tokio-util = { version = "0.7", features = ["compat"] }
```

Comment:
We should use the workspace version: `tokio-util.workspace = true`.

But I don't think we actually use `tokio-util` at all here, so we can just drop it.